code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -0,0 +1,395 @@
+	trigger_range = 0
+	ignored_factions = list("ecult")
+	range = 15
+	speed = 1
+	var/static/list/blacklisted_turfs = typecacheof(list(/turf/closed,/turf/open/space,/turf/open/lava,/turf/open/chasm,/turf/open/floor/plating/rust))
+
+/obj/projectile/magic/spell/rust_wave/Moved(atom/OldLoc, Dir)
+	. = ..()
+	var/list/turflist = list()
+	var/turf/T1
+	turflist += get_turf(src)
+	switch(Dir)
+		if(NORTH)
+			T1 = get_step(src,WEST)
+			turflist += T1
+			turflist += get_step(T1,WEST)
+			T1 = get_step(src,EAST)
+			turflist += T1
+			turflist += get_step(T1,EAST)
+		if(SOUTH)
+			T1 = get_step(src,WEST)
+			turflist += T1
+			turflist += get_step(T1,WEST)
+			T1 = get_step(src,EAST)
+			turflist += T1
+			turflist += get_step(T1,EAST)
+		if(WEST)
+			T1 = get_step(src,NORTH)
+			turflist += T1
+			turflist += get_step(T1,NORTH)
+			T1 = get_step(src,SOUTH)
+			turflist += T1
+			turflist += get_step(T1,SOUTH)
+		if(EAST)
+			T1 = get_step(src,NORTH)
+			turflist += T1
+			turflist += get_step(T1,NORTH)
+			T1 = get_step(src,SOUTH)
+			turflist += T1
+			turflist += get_step(T1,SOUTH)
+
+	for(var/X in turflist)
+		var/turf/T = X
+		if(!is_type_in_typecache(T, blacklisted_turfs) && prob(75))
+			T.ChangeTurf(/turf/open/floor/plating/rust)
+
+/obj/effect/proc_holder/spell/targeted/projectile/dumbfire/rust_wave/short
+	name = "Small Patron's Reach"
+	proj_type = /obj/projectile/magic/spell/rust_wave/short
+
+/obj/projectile/magic/spell/rust_wave/short
+	range = 7
+	speed = 2
+
+/obj/effect/proc_holder/spell/pointed/ash_cleave
+	name = "Cleave"
+	desc = "Causes severe bleeding on a target and people around him"
+	school = "transmutation"
+	charge_max = 350
+	clothes_req = FALSE
+	invocation = "CL'VE"
+	invocation_type = "whisper"
+	range = 9
+	action_icon = 'icons/mob/actions/actions_ecult.dmi'
+	action_icon_state = "ash_shift"
+	action_background_icon_state = "bg_ecult"
+
+/obj/effect/proc_holder/spell/pointed/ash_cleave/cast(list/targets, mob/user)
+	if(!targets.len)
+		to_chat(user, "<span class='warning'>No target found in range!</span>")
+		return FALSE
+	if(!can_target(targets[1], user))
+		return FALSE
+
+	for(var/mob/living/carbon/human/C in range(1,targets[1]))
+		targets += C
+
+	for(var/X in targets)
+		var/mob/living/carbon/human/target = X
+		if(target == user)
+			continue
+		if(target.anti_magic_check())
+			to_chat(user, "<span class='warning'>The spell had no effect!</span>")
+			target.visible_message("<span class='danger'>[target]'s veins flash with fire, but their magic protection repulses the blaze!</span>", \
+							"<span class='danger'>Your veins flash with fire, but your magic protection repels the blaze!</span>")
+			continue
+
+		target.visible_message("<span class='danger'>[target]'s veins are shredded from within as an unholy blaze erupts from their blood!</span>", \
+							"<span class='danger'>Your veins burst from within and unholy flame erupts from your blood!</span>")
+
+		target.bleed_rate += 8
```

Comment:
whoop de fuckin' do

you do know that bleed_rate decays naturally, right?

Reply:
i will look into it